docs/docs/configuration.md

```diff
@@ -203,6 +203,23 @@ This can be useful if you are hosting a private feed and need to host large pack
 }
 ```
 
+## Statistics
```

Comment:
This section needs to be sdjusted to the latest changes.

Reply:
Yeah we have to merge the sections, right?

Reply:
I thought more of adjusting the given example :-)